client/my-sites/site-settings/form-writing.jsx

```diff
@@ -126,7 +126,7 @@ class SiteSettingsFormWriting extends Component {
 				}
 
 				{
-					this.renderSectionHeader( translate( 'Custom content types' ) )
+					this.renderSectionHeader( translate( 'Content types' ) )
```

Comment:
The existing translations are for an obsolete string (a bit of an edge case here), so nothing to change elsewhere. We should be able to reuse the existing translations when this is merged.